studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,691 @@
+    '?graphRef=Apollo-Fullstack-Demo-o3tsz8@current' +
+    '&postMessageOperations=true';
+
+  return (
+    <div className="App">
+      <h1 className="demo-app-title">Apollo's Embedded Explorer Demo App</h1>
+      <iframe id="embedded-explorer" className="embedded-explorer-iframe" title="embedded-explorer" src={explorerURL} />
+    </div>
+  );
+}
+```
+
+</MultiCodeBlock>
+
+#### 5. Run the example
+
+You can view and run a complete example that uses code very similar to the above on Codesandbox:
+
+<a href="https://codesandbox.io/s/github/apollographql/embeddable-explorer-demo-react/tree/main/?fontsize=14&hidenavigation=1&theme=dark">
+  <img alt="Edit embedded-explorer-demo" src="https://codesandbox.io/static/img/play-codesandbox.svg" />
+</a>
+
+The example includes support for subscriptions, which are omitted from this Setup section for clarity. The subscription-specific logic should be more straightforward after familiarizing yourself with the logic for queries and mutations.
+
+### Message types
+
+If you're using the `postMessage` API with your embedded Explorer, every message posted this way has an `event.data.name` which indicates the type of message.
+
+These are the message types used by the embedded Explorer's `iframe` and your page:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `ExplorerRequest`
+
+</td>
+<td>
+
+Posted by the `iframe` to notify your page that the user wants to execute a query or mutation.
+
+The following operation details are included in the associated `event.data` object:
+
+* `operation` (string)
+* `operationId` (string)
+* `operationName` (string)
+* `variables` (object)
+* `headers` (object)
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `ExplorerResponse`
+
+</td>
+<td>
+
+Posted by your page to notify the `iframe` of a query or mutation result. The message should be the operation's JSON response object.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `ExplorerSubscriptionRequest`
+
+</td>
+<td>
+
+Posted by the `iframe` to notify your page that the user wants to execute a subscription operation.
+
+The same operation details are included in the `event.data` object that are included in [`ExplorerRequest` messages](#explorerrequest).
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `ExplorerSubscriptionResponse`
+
+</td>
+<td>
+
+Posted by your page to notify the `iframe` of a new subscription result. The message should be the subscription's new JSON response object.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `ExplorerSubscriptionTermination`
+
+</td>
+<td>
+
+Posted by the `iframe` to notify your page that the user wants to terminate an active subscription operation.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `SetOperation`
+
+</td>
+<td>
+
+Posted by your page to provide the `iframe` with an operation string to populate the Explorer's editor. You might want to use this to populate a default operation on page load.
```

Comment:
Now that we have tabs, this will currently replace the last open tab. I am changing it to open a new tab, but not sure if the copy needs to change for tabs. (I think its fine?)